sandbox

```diff
@@ -224,6 +224,37 @@ sandbox () {
     fi    
   }
 
+  # wait until indexer/algod health endpoints report 200
```

Comment:
@winder Sanity checking:  Is it typical to run shellcheck on scripts in _sandbox_?

As part of my review, I ran shellcheck and discovered a number of warnings that predate the PR.  So, I'm guessing it's _not_ typically invoked though I want to confirm.

I'm on-board to leave approval as is - I successfully tested the PR.  While I don't want to hold up the PR, I'm doing my due diligence as a reviewer here.

Reply:
I had a verbal with @winder where it was confirmed that `sandbox` is known to fail shellcheck, and passing shellcheck is _not_ in the PR's scope.  Resolving.

Reply:
Thanks for the reminder, I fixed the shellcheck warnings introduced by this PR.